src/site/markdown/server/manage-webadmin.md

```diff
@@ -736,6 +736,18 @@ The message is UNSEEN via IMAP
 
 [More details about endpoints returning a task](#Endpoints_returning_a_task).
 
+An admin can specify the concurrency that should be used when running the task:
+
+ - `messagesPerSecond` rate at which messages should be processed, per second. Defaults to 100.
```

Comment:
s/ rate at which messages should be processed, per second/ rate of messages to be processed per second
s/Defaults/Default